app/policies/rent_policy.rb

```diff
@@ -0,0 +1,23 @@
+class RentPolicy < ApplicationPolicy
+  attr_reader :user, :rent
+  def initialize(user, rent)
+    @user = user
+    @rent = rent
+  end
+
+  def create?
+    @rent.user_id == user.id
+  end
+  class Scope
+    attr_reader :user, :scope
+
+    def initialize(user, scope)
+      @user = user
+      @scope = scope
+    end
+
+    def resolve
+      scope.select { |rent| rent.user.id == user.id }
```

Comment:
here you are loading *every* item and filtering on ruby memory. Instead you can use `.where(user: user)`, which filters using a `where` SQL statement instead of using ruby